tests/ReviewsApiTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+use Rogue\Models\Event;
+use Rogue\Models\Photo;
+use Rogue\Models\Post;
+use Faker\Generator;
+
+class ReviewsApiTest extends TestCase
+{
+    /*
+     * Base URL for the Api.
+     */
+    protected $reviewsApiUrl = 'api/v2/reviews';
+
+    /**
+     * Test that a PUT request to /reviews updates the status and creates a new event and review.
+     *
+     * PUT /reviews
+     * @return void
+     */
+    public function testUpdatingASingleReview()
+    {
+        // Create a photo.
+        $photo = factory(Photo::class)->create();
+
+        // Create a post and associate the photo with it.
```

Comment:
I wonder if we should move this work into a model event also. So when a type of thing is created we also automatically create the post for it. Not needed for merge, just a thought.